graphql_compiler/tests/test_helpers.py

```diff
@@ -744,7 +779,11 @@ def generate_schema_graph(orientdb_client):
     return get_orientdb_schema_graph(schema_data, index_query_data)
 
 
-def generate_schema(orientdb_client, class_to_field_type_overrides=None, hidden_classes=None):
+def generate_schema(
+    orientdb_client: OrientDB,
+    class_to_field_type_overrides: Optional[ClassToFieldTypeOverridesType] = None,
+    hidden_classes: Optional[Set[str]] = None,
+) -> Tuple[GraphQLSchema, Dict[Union[GraphQLInterfaceType, GraphQLObjectType], GraphQLUnionType]]:
```

Comment:
I am sorry for not mentioning this earlier, but I think it might be worthwhile to create a `TypeEquivalenceHintsType` too. We are probably going to be redefining it a bunch of times otherwise.